temporal-test-server/src/main/java/io/temporal/internal/testservice/TestWorkflowService.java

```diff
@@ -1027,6 +1031,11 @@ public void getDiagnostics(StringBuilder result) {
     store.getDiagnostics(result);
   }
 
+  /**
+   * @deprecated use {@link io.temporal.serviceclient.TestServiceStubs} and {@link
+   *     io.temporal.api.testservice.v1.TestServiceGrpc.TestServiceBlockingStub#getCurrentTime(Empty)}
+   */
+  @Deprecated
```

Comment:
Are users using these directly and that's why we need to deprecate them?

Reply:
They shouldn't, because it's in `internal` package and has comments to don't use it directly. But I know some users who were playing with the test server quite a bit maybe using and exposing them in their own wrappers, yes.
I'm going to clean this up next time I touch code around it (even if it's the next release)